src/components/spinnerV2/spinnerV2.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { translate } from 'react-i18next';
+import styles from './spinnerV2.css';
+
+const SpinnerV2 = ({ t }) => (
+  <span className={styles.wrapper}>
+    <span className={`${styles.spinner} spinner`}/>
+    <span className={`${styles.label}`}>{t('Pending...')}</span>
```

Comment:
I think `t('Pending...')` should be passed from outside as `label` prop, since it can be other text in other places.